samplestore/src/main/java/com/stripe/samplestore/RetrofitFactory.java

```diff
@@ -0,0 +1,47 @@
+package com.stripe.samplestore;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava.RxJavaCallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Factory instance to keep our Retrofit instance.
+ */
+public class RetrofitFactory {
+
+    // Put your Base URL here. Unless you customized it, the URL will be something like
+    // https://hidden-beach-12345.herokuapp.com/
+    private static final String BASE_URL = "Put your server base path here.";
```

Comment:
Is it normal to put configuration options in the code as hardcoded values?  Should this be coming from a config.xml or something containing the values?  Just not sure what best practice is here.